community_images/apache/bitnami/.rfignore

```diff
@@ -0,0 +1,2 @@
+opt/bitnami/apache/include
+opt/bitnami/licenses
```

Comment:
There were many other folders which needed to be mentioned in .rfignore, right?

Reply:
Those folders are not directly related to the apache server but are meant for the system functionality.

Reply:
Noted @Rajat0002, thanks. Can you please check and add these directories here as well?
opt/bitnami/common/licenses
opt/bitnami/apache2/licenses or opt/bitnami/apache/licenses (Whichever exists)
usr/share/common-licenses

Reply:
@AnmolVirdi directories added